metricbeat/docs/developer-guide/metricset-details.asciidoc

```diff
@@ -147,7 +147,8 @@ We recommend that you use all three when you create a metricset. Unit tests are
 written in Golang and have no dependencies. Integration tests are also written
 in Golang but require the service from which the module collects metrics to also be running.
 System tests for Metricbeat also require the service to be running in most cases and are
-written in Python based on our small Python test framework.
+written in Python based on our small Python test framework. We use `virtualenv` to deal with Python dependences,
```

Comment:
This sentence is grammatically ambiguous. Is this what you mean to say? [note the typo fix, too...should say dependencies not dependences.]

```
We use `virtualenv` to deal with Python dependencies. You can simply run the command `make python-env` 
and then `. build/python-env/bin/activate` . 
```